### PR TITLE
Update installation md

### PR DESCRIPTION
**Pull request by rpottsoh**

### CODE REVIEW

File: docs/INSTALLATION.md [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
+A 30 day free trial of Embarcadero's RAD Studio or Delphi may be downloaded from [here](https://www.embarcadero.com/products?utm_source=google&utm_medium=cpc&utm_campaign=brand&utm_content=brand&utm_embarcadero&gclid=CjwKEAiAp97CBRDr2Oyl-faxqRMSJABx4kh9V8bOEuG0CznQ9AGToIyuKeTzvevljmHTboYXk4n6OxoC4Frw_wcB).  Alternatively, a free [Starter Edition](https://www.embarcadero.com/products/delphi/starter/promotional-download) version is also available.  Follow installation instructions included with the product.
 
 ### Non-Windows based Operating Systems ###
 
-Sorry, Delphi does not run natively on any other operating systems.  However it should be possible to install Delphi into a virtual machine which is running Windows.
+Delphi does not run natively in non-Windows based operating systems.  However, it is possible to install Delphi into a virtual machine which is running Windows.  If utilizing a Virtual Machine it is possible for Delphi to function.
```
Comment (kotp) [inline docs/INSTALLATION.md]: The sentence is redundant.  Combine it, as I think you are trying to say something to the effect that "It is possible to install Delphi on a virtual machine with the client operating system being Windows, and Delphi has been found to work fine in this case."

Even perhaps suggest virtual box here.

File: docs/INSTALLATION.md [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
+
+With Delphi successfully installed and working correctly please follow these steps for retrieving and installing DUnitX if you find that your version of Delphi is lacking the DUnitX framework.
 
 - DUnitX is an open source project by [VSoftTechnologies](https://www.github.com/VSoftTechnologies).  The DUnitX github repo is available [here](https://github.com/VSoftTechnologies/DUnitX).
 - It is best to fork the repo and then clone or otherwise download the fork to your local machine.
 - Step-by-step instructions on how to install DUnitX into Delphi have been posted by Vincent Parrett in a blog post located [here](https://www.finalbuilder.com/resources/blogs/postid/702/dunitx-has-a-wizard).
+- The Starter Edition of Delphi doesn't allow the use of project experts.  DUnitX comes with an expert which will aid the student in creating a new DUnitX based project.  The expert creates the projects .DPR file.  The exercises in this track all include the .DPR file making the expert not necessary.  DUnitX otherwise has no issues with projects created in Delphi version starting with XE all the up to and including the Starter Edition of Berlin.
```
Comment (kotp) [inline docs/INSTALLATION.md]: Nice information and distinction here.

File: exercises/saddle-points/uSaddlePointsExample.pas [hunk 3]
```diff
@@ -1,3 +1,14 @@
+//========================================================================
```
Comment (kotp) [inline exercises/saddle-points/uSaddlePointsExample.pas]: This file doesn't belong in this PR.

File: exercises/saddle-points/uSaddlePointsTests.pas [hunk 4]
```diff
@@ -1,3 +1,12 @@
+//******************************************************************
```
Comment (kotp) [inline exercises/saddle-points/uSaddlePointsTests.pas]: This file doesn't belong in this PR.

File: exercises/saddle-points/uSaddlePointsExample.pas (hunk 3 above)
Comment (rpottsoh) [inline exercises/saddle-points/uSaddlePointsExample.pas]: This is where I am having difficulties with PRs and squash & merge.  When I open a new PR ALL the commits from other PRs are listed.  I've assumed that when I squash merge a PR all the commits contained within the PR go away.  I am only seeing the list of commits growing, very annoying and confusing.

File: docs/INSTALLATION.md (hunk 1 above)
Comment (rpottsoh) [inline docs/INSTALLATION.md]: I'll work on this, it is a clumsy statement at best and I see another minor issue I want to address as well.

File: exercises/saddle-points/uSaddlePointsExample.pas (hunk 3 above)
Comment (kotp) [inline exercises/saddle-points/uSaddlePointsExample.pas]: Let's spend some time together IRL when you get some free time.
